WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -236,15 +235,17 @@ public void handleEditPostResult(int resultCode, Intent data) {
         }
 
         UploadUtils.handleEditPostResultSnackbars(getActivity(),
```

Comment:
Just wondering - can't the post be null if `post == null` and `EXTRA_IS_DISCARDABLE` is true?
If it can't be null, I'd suggest removing the `(post == null)` check. Wdyt?

Reply:
I guess I simply assumed whoever wrote the original code had a reason for checking for null :) I _think_ it's possible it could be null, in which case there's no good reason to show a snackbar in the first place. I updated the code in f87224e to handle this.